src/Build/Evaluation/Expander.cs

```diff
@@ -2264,15 +2333,12 @@ internal static ItemTransformFunction GetItemTransformFunction(IElementLocation
                         }
                     }
 
-                    // Phase 2: Go through the directories and look for the specified file.
+                    // Phase 2: Go through the directories and return them in order
```

Comment:
Think we should specify, guarantee, and test this order?

Reply:
This is more for my own sanity than anything else, so... not really. :-)

Reply:
When we document these we should remember to say "don't depend on the order", just in case.